packages/inventory-compliance/src/__snapshots__/SystemRulesTable.test.js.snap

```diff
@@ -30,7 +30,7 @@ exports[`SystemRulesTable component should render 1`] = `
             onButtonClick={[Function]}
             onFilterChange={[Function]}
             onOptionSelect={[Function]}
-            placeholder="Search by name or identifer"
+            placeholder="Search by name, identifer, or reference"
```

Comment:
This will get cut off on narrow screens. It was already being cut off before this change. @katierik I think this is a UX issue

Reply:
Ick, that is no good. I don't have an immediate solution for this with the filter. The attribute value really isn't set up for multiple things.
We could break it into 3 filters: name; identifiers; references; but that is non-ideal. 
I'll bring this up with PF and also see if I can come up with a better solution.

Reply:
FOLLOW UP: For now, we can increase the max width of the filter box to see more of the content